webview/src/experiments/components/table/body/Row.tsx

```diff
@@ -107,8 +104,7 @@ export const RowContent: React.FC<
             [styles.evenRow]: !isOdd,
             [styles.workspaceRow]: isWorkspace,
             [styles.normalRow]: !isWorkspace,
-            [styles.rowSelected]: isRowSelected,
-            [styles.rowFocused]: menuActive
```

Comment:
this `.rowFocused` style doesn't exist which means we don't need `menuActive` which means we no longer need `onShow` and `onHide`